src/libraries/System.Private.CoreLib/src/System/Span.cs

```diff
@@ -38,6 +38,7 @@ namespace System
         /// <param name="array">The target array.</param>
         /// <remarks>Returns default when <paramref name="array"/> is null.</remarks>
         /// <exception cref="ArrayTypeMismatchException">Thrown when <paramref name="array"/> is covariant and array's type is not exactly T[].</exception>
+        [NonVersionable]
```

Comment:
Hypothetically, would we ever change handling of array covariance? The constructor doesn't look trivial enough.

Reply:
Changing it would require a R2R version bump anyway due to the indexer being an intrinsic.